sonar-plugin-api/src/main/java/org/sonar/api/resources/Qualifiers.java

```diff
@@ -46,7 +46,9 @@ private Qualifiers() {
   /**
    * Library, for example a JAR dependency of Java projects.
    * Scope of libraries is Scopes.PROJECT
+   * @deprecated since 5.2 No more design features
    */
+  @Deprecated
   public static final String LIBRARY = "LIB";
```

Comment:
:new: ![INFO](https://raw.githubusercontent.com/henryju/image-hosting/master/info.png)Do not forget to remove this deprecated code someday.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)